assets/components/svgs/passwordKey.jsx

```diff
@@ -0,0 +1,14 @@
+// @flow
+
+import React from 'react';
+
+// A padlock icon used to signify that checkouts are secure.
```

Comment:
why not call it `padlock` then? 😄

Reply:
interestingly, apparently padlocks in the body of the page are more trusted than padlocks in the browser furniture 🤦‍♂️

Reply:
ahhhh this is so annoying. I fixed these and pushed them but as I'd done commit ammend they hadn't come through. Sorry for wasting your time! (Same goes to the console log below @tsop14 )

Reply:
this isn't a padlock, it's a key